### PR TITLE
Remove a duplicated piece of JS

### PR DESCRIPTION
**Pull request by pypingou**
Signed-off-by: Pierre-Yves Chibon <pingou@pingoured.fr>

### CODE REVIEW

**Comment by pypingou** [comment]
Jenkins trips over: ````rawhide	pkg_resources.DistributionNotFound: The 'pyflakes<1.7.0,>=1.5.0' distribution was not found and is required by flake8```` but the tests are passing otherwise

File: bodhi/server/templates/update.html [hunk 1]
```diff
@@ -51,20 +51,8 @@
...
-                if (missing_tests[req.item] === undefined)
-                    missing_tests[req.item]= [];
-                missing_tests[req.item].push(req);
-            }
-            // the user may have already specified this in the required taskotron tests
-            if ($.inArray(req.testcase, requirements) == -1) {
-                requirements.push(req.testcase);
-            }
-        });
-    };
```
Comment (bowlofeggs) [inline bodhi/server/templates/update.html:67]: Whoah that is strange. I wonder how I didn't notice this before...‽

**Comment by bowlofeggs** [comment]
jenkies test

**Review by bowlofeggs — APPROVED** [review]

**Comment by ralphbean** [comment]
Doh, merge conflict with #2393.

**Pull request merged**